### DISCUSSION TITLE
REST API not working

### DISCUSSION THREAD

**Discussion by kbxian** [Tmeister/wp-api-jwt-auth#267] [Q&A]
Hi, I am having trouble calling the REST API through https://website.com/wp-json/jwt-auth/v1/token2

It's returning this: {"code":"rest_no_route","message":"No route was found matching the URL and request method.","data":{"status":404}}

Could it be related to JWT Authentication plugin?

**Comment by AhmedAshraf0** [comment]
Hey i'm facing the same problem now -_-
Did you find what to do?

**Reply by AhmedAshraf0** [reply]
I found it: https://example.com/wp-json/api/v1/token
and pass the username and the password as query parameters.
Then you should receive the json response with the token in it.
Got it from: https://www.youtube.com/watch?v=XlbSVHR7ohQ&ab_channel=miniOrange
